release/__init__.py

```diff
@@ -64,8 +63,7 @@ def expand_env_vars(config):
 
 
 def load_config(filename):
-    with open(filename) as config_file:
-        return expand_env_vars(yaml.load(config_file))
+    return expand_env_vars(pkgpanda.util.load_yaml(filename))
```

Comment:
pkgpanda.util is/was already imported and this function isn't used much. My general rule is that unless it's used a number of times (3+) then doing the whole name-spacing in the callsite reduces the amount of work it takes a reader to follow the code back to source (Someone reading the code sees a called function from another module). If it's used a lot then there's an advantage gained from typing less by making a local alias (what from import does). In this case the from import would actually be more code.